package.json

```diff
@@ -50,56 +52,59 @@
   },
   "dependencies": {
     "@0x/contracts-utils": "3.1.1",
-    "abi-decoder": "^2.2.0",
```

Comment:
I will lock the versions in package.json, thank you very much for your opinion.
There is a moronically retarded pattern amongst npm module maintainers to expose directly the dependencies' API as their own, which we come to rely on, and then when they change the minor version of their package change the subdependencies' major versions, which break. I am not ever interested in a range of versions, fuck that. I want the exact version I am testing against forever until I tell the computer to change that and not the other way around.

Reply:
you say that NPM is morons. maybe, but that's what we have to use, and we can't halfway use different pattern.
the standard pattern is a "lock" file, so we use that, and don't invent the wheel
 ^ means "upgrade" works on minor releases (and if you have a lock file, it doesn't change)
if some package misbehave - lock on THAT package, don't attempt at "all" (since you modify only  ~30 of them)
(just checked: we use 1054 unique packages, and  3085 versions)